### PR TITLE
Add info tooltip for metadata search

### PR DESCRIPTION
**Pull request by balanza**
# Description

Help users to discover search syntax for activity logs.

### CODE REVIEW

File: assets/js/pages/ActivityLogPage/ActivityLogPage.jsx [hunk 1]
```diff
@@ -174,15 +174,27 @@ function ActivityLogPage() {
...
 
+  const searchInfo = (
+    <div className="text-center">
+      Filter by 1 or more keywords.
+      <br />
+      Refine filters by using &apos;AND&apos; & &apos;OR&apos;.
+      <br />
+      e.g: created OR login.
+      <br />
+      For more <a href="#docs">view our docs</a>.
```
Comment (balanza) [inline assets/js/pages/ActivityLogPage/ActivityLogPage.jsx:185]: @jagabomb we neet the real link, can you help?

Comment (balanza) [reply assets/js/pages/ActivityLogPage/ActivityLogPage.jsx:185]: Leaving the PR as draft meanwhile

Comment (balanza) [reply assets/js/pages/ActivityLogPage/ActivityLogPage.jsx]: Nevermind, let's add the link in a next PR

**Comment by jagabomb** [comment]
@balanza Do you have a screenshot for this PR?
I have tried looking at the [PR environment](https://3186.prenv.trento.suse.com/activity_log) and Storybook but I can't see any changes to the [Activity Log Message Field](https://643fc66e9747e47344899b8c-monmlhzgjc.chromatic.com/?path=/story/components-activitylogoverview--default).
Would it be better to pull the PR and run locally? 😅

**Comment by balanza** [comment]
> @balanza Do you have a screenshot for this PR? I have tried looking at the [PR environment](https://3186.prenv.trento.suse.com/activity_log) and Storybook but I can't see any changes to the [Activity Log Message Field](https://643fc66e9747e47344899b8c-monmlhzgjc.chromatic.com/?path=/story/components-activitylogoverview--default). Would it be better to pull the PR and run locally? 😅

@jagabomb You won't see anything on the storybook 'cause changes are in the page

image

**Comment by jagabomb** [comment]
Thanks for the assistance @balanza and @nelsonkopliku. I can see it now... it looks good so far. Did you try look into the possibility of adding the **Clear** button for the message field next to the tooltip icon? I remember this might have been tricky but I just want to check if we tried it.

@abravosuse and @nelsonkopliku, are you still happy with the message in the tooltip? I just want to confirm if you think it explains the functionality well enough?

**Comment by nelsonkopliku** [comment]
@jagabomb wrt the clear action, at the end seems to be feasible without much hassle.
We can keep `allowClear: true` prop and add an `inline` class to the `EOS_INFO_OUTLINED` icon in the tooltip.'

image (1)

With regards to the message I am fine with it.

**Comment by jagabomb** [comment]
> @jagabomb wrt the clear action, at the end seems to be feasible without much hassle. We can keep `allowClear: true` prop and add an `inline` class to the `EOS_INFO_OUTLINED` icon in the tooltip.'
> 
> image (1)
> 
> With regards to the message I am fine with it.

Amazing, I like this solution because it keeps the UX consistent with the other components on the page. Cool, thanks for

**Review by nelsonkopliku — APPROVED** [review]

**Comment by balanza** [comment]
> > @jagabomb wrt the clear action, at the end seems to be feasible without much hassle. We can keep `allowClear: true` prop and add an `inline` class to the `EOS_INFO_OUTLINED` icon in the tooltip.'
> > image (1)
> > With regards to the message I am fine with it.
> 
> Amazing, I like this solution because it keeps the **clear field** functionality with the other components on the page. Cool, thanks for confirming the text for the tooltip.

Obbedisco :saluting_face: 
image

**Pull request merged**